components/bl-AnimateOnScroll-component/component.json

```diff
@@ -0,0 +1,260 @@
+{
+  "id": "c_10781b7f3c7d301c2459726acab35b51",
+  "name": "Animate on Scroll",
+  "description": "This component allows adding animation for elements when scrolling the page",
+  "showInToolbox": true,
+  "faIcon": "caret-square-down",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "select",
+      "name": "disable",
+      "label": "Disable",
+      "showInSettings": true,
+      "defaultValue": "false",
+      "hasLogicHandler": true,
+      "handlerId": "disableLogic",
+      "handlerLabel": "Disable Logic",
+      "dataBinding": true,
+      "options": [
```

Comment:
Do we handle case when user returns boolean true from this handler(as it works in all other components) ?